### PR TITLE
Fix marginalization during prediction

### PR DESCRIPTION
**Pull request by dusenberrymw**
Marginalization should be performed on an float32 image with values in
the interval [0,1], followed by normalization.

### CODE REVIEW

**Comment by dusenberrymw** [comment]
@feihugis can you try this out?

File: breastcancer/inference.py [hunk 1]
```diff
@@ -194,8 +194,9 @@ def predict_mitoses_num_locations(model, model_name, threshold, ROI, tile_size=6
     for tile in tiles:
       # NOTE: averaging over sigmoid outputs vs. logits may yield slightly different results, due
       # to numerical precision
-      norm_tile = normalize((tile / 255).astype(np.float32), model_name)  # normalize tile
-      aug_tiles = create_augmented_batch(norm_tile, batch_size)  # create batch of aug versions
+      prep_tile = (tile / 255).astype(np.float32)  # convert to values in [0,1]
+      aug_tiles = create_augmented_batch(prep_tile, batch_size)  # create batch of aug versions
+      norm_tiles = normalize(aug_tiles, model_name)  # normalize augmented tiles
       aug_preds = model(aug_tiles)  # make predictions on augmented batch
       pred = marginalize(aug_preds)  # average predictions
```
Comment (feihugis) [inline breastcancer/inference.py]: Could these variables be moved out of the loop? Otherwise, they will be repeatedly added into the graph and make the program run slower.

**Comment by feihugis** [comment]
@dusenberrymw LGTM!
